actors/builtin/miner/miner_actor.go

```diff
@@ -1350,7 +1350,7 @@ func (a Actor) AddLockedFund(rt Runtime, amountToLock *abi.TokenAmount) *adt.Emp
 			rt.Abortf(exitcode.ErrInsufficientFunds, "insufficient funds to lock, available: %v, requested: %v", availableBalance, *amountToLock)
 		}
 
-		if err := st.AddLockedFunds(store, rt.CurrEpoch(), *amountToLock, &PledgeVestingSpec); err != nil {
+		if err := st.AddLockedFunds(store, rt.CurrEpoch(), *amountToLock, &RewardVestingSpec); err != nil {
```

Comment:
It seems reasonable to impose reward vesting on funds miners intentionally lock up, but perhaps there is something more sophisticated we should be doing like enforcing different specs for different callers.

Reply:
I think the prior is how cryptoecon want it. The reward is considered pledged to the network, it's just not the "initial pledge" for a sector. I think there's only one PledgeVestingSpec.

Reply:
It seems I was wrong. They actually would prefer something like #537, but for today, different specs are good.